README.md

```diff
@@ -17,8 +17,43 @@ This plugin integrates the ServiceNow Virtual Agent in Mattermost. For a stable
 
 ## Features
 
-This plugin supports sending text messages to the Virtual Agent through Mattermost and handling/displaying different types of responses from the Virtual Agent.
-**Note-** Currently we only support sending text messages and file attachments and displaying text, picker/dropdown and link responses from the Virtual Agent API.
+- ### This plugin supports sending the below fields to the Virtual Agent through Mattermost-
+  1. **Text messages**
+
+        ![image](https://user-images.githubusercontent.com/55234496/196138915-9577372f-89a4-49f7-bc02-398ca42a7003.png)
```

Comment:
![image](https://user-images.githubusercontent.com/77336594/196499204-9658ac8e-3431-4a63-b16b-49371ea3cfa5.png)

What's the need for hiding the username here?